ADAL/src/ADAuthenticationContext.m

```diff
@@ -134,6 +134,7 @@ - (id)initWithAuthority:(NSString *)authority
 - (ADAuthenticationRequest*)requestWithRedirectString:(NSString*)redirectUri
                                              clientId:(NSString*)clientId
                                              resource:(NSString*)resource
+                                   clientCapabilities:(NSArray<NSString *> *)clientCapabilities
                                       completionBlock:(ADAuthenticationCallback)completionBlock
```

Comment:
Just curious, why do we need to make `clientCapabilities` a parameter for the function given that capabilities could be accessed by `_clientCapabilities`? :)
Besides, I am also not sure why we need `redirectUri`, `clientId` and `resource` as parameters for this function, haha